proposals/working-group/microprofile-working-group/sponsorhsip.adoc

```diff
@@ -0,0 +1,31 @@
+= Fee Structure Option
+One of the main barriers to a specification process is the cost associated with joining. Companies
```

Comment:
Something's strange with this file...  When I look a rendered version of this .adoc, I get the title (Fee Structure Option), but then the text starts with line 4 (to pay a fee...).